rest/src/main/scala/io/udash/rest/openapi/OpenApiMetadata.scala

```diff
@@ -52,6 +52,16 @@ final case class OpenApiMetadata[T](
 ) {
   val httpMethods: List[OpenApiOperation[_]] = (gets: List[OpenApiOperation[_]]) ++ customBodyMethods ++ bodyMethods
 
+  // collect all tags
+  private lazy val openApiTags: List[Tag] = {
+    def createTags(method: OpenApiMethod[_]): List[Tag] =
+      method.groupAnnot.fold[List[Tag]](Nil) { group =>
+        List(method.tagAdjusters.foldLeft(Tag(group.groupName))({ case (tag, adjuster) => adjuster.adjustTag(tag) }))
+      }
+
+    prefixes.flatMap(prefix => createTags(prefix) ++ prefix.result.value.openApiTags) ++ httpMethods.flatMap(createTags)
```

Comment:
You know I'm not a fan of concatenating lists and I'd rather use iterators here to avoid intermediate collections and additional traversal.